.github/workflows/benchmark.yml

```diff
@@ -86,10 +86,50 @@ jobs:
           cache-to: type=gha,mode=max
           build-args: DIST=build
           target: app
+  build-benchmark-images:
+    name: Build starter and worker images
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    outputs:
+      image-tag: ${{ steps.image-tag.outputs.image-tag }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref }}
+      - uses: google-github-actions/auth@v1
+        id: auth
+        with:
+          token_format: 'access_token'
+          workload_identity_provider: 'projects/628707732411/locations/global/workloadIdentityPools/zeebe-gh-actions/providers/gha-provider'
```

Comment:
:thinking: Wondering whether this is necessary, since we already did this above? Is it because it is a different job?

Reply:
Yep, that's it